composer.json

```diff
@@ -15,6 +15,9 @@
     ],
     "require": {
         "php": ">=5.3.6",
+        "ext-ctype": "*",
```

Comment:
it may? indirect?
If our package use this ext directly it should be listed, if not - then not.